pkgs/top-level/all-packages.nix

```diff
@@ -30812,11 +30812,23 @@ with pkgs;
   firefox-esr-unwrapped = firefoxPackages.firefox-esr-102;
 
   firefox = wrapFirefox firefox-unwrapped { };
-  firefox-beta = wrapFirefox firefox-beta-unwrapped { };
-  firefox-devedition = wrapFirefox firefox-devedition-unwrapped { };
+  firefox-beta = wrapFirefox firefox-beta-unwrapped {
+    nameSuffix = "-beta";
+    desktopName = "Firefox Beta";
+    wmClass = "firefox-beta";
+  };
+  firefox-devedition = wrapFirefox firefox-devedition-unwrapped {
+    nameSuffix = "-devedition";
+    desktopName = "Firefox DevEdition";
+    wmClass = "firefox-devedition";
+  };
 
   firefox-esr = firefox-esr-102;
-  firefox-esr-102 = wrapFirefox firefox-esr-102-unwrapped { };
+  firefox-esr-102 = wrapFirefox firefox-esr-102-unwrapped {
+    nameSuffix = "-esr";
+    desktopName = "Firefox ESR";
+    wmClass = "firefox-esr";
```

Comment:
We usually have two ESR releases for 3 months or so every once in a while, when they switch between one and the next.

Reply:
I think the best way to handle this is to make firefox-esr a symlink to firefox-esr-102 but it is not obvious to me about how to do this in nix.

Maybe we can leave it for later because it will not be a breaking change.